datastore/README.md

```diff
@@ -0,0 +1,59 @@
+# Cloud Datastore Sample
+
+This sample provides example code for
+[cloud.google.com/datastore/docs](https://cloud.google.com/datastore/docs).
+
```

Comment:
Let's mention that the Cloud Datastore API needs to be enabled here...

> Before you can run or deploy the sample, you will need to enable the Cloud Datastore API in the [Google Developers Console](https://console.developers.google.com/project/_/apiui/apiview/datastore/overview).

For example: [Python Datastore README](https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/managed_vms/datastore#setup)